utils/not_doctested.txt

```diff
@@ -371,7 +371,6 @@ src/transformers/generation/beam_constraints.py
 src/transformers/generation/beam_search.py
 src/transformers/generation/flax_logits_process.py
 src/transformers/generation/flax_utils.py
-src/transformers/generation/logits_process.py
```

Comment:
@ydshieh this is enough to make the file part of the doctests, correct? :)

Reply:
Yes!